testing/tests/test_transformers.py

```diff
@@ -0,0 +1,118 @@
+import filecmp
+import json
+import perftestnotebook
+import pytest
+import yaml
+from utilities import get_nested_values
+from functools import reduce
+from operator import add
+from pathlib import Path
+
+import pytest
+
+
+class TestTransformer(object):
+    def check_transformation(self, test_config_filepath, target_config_filepath, get_nested_keys):
+        """
+        This function checks:
+            - Both of test and local config files from the arguments have "file_groups" field.
```

Comment:
nit: Should be `That both testing and local config files from the arguments have "file_groups" field.`

Also, can you add some inline comments in the test to make where all these tests are happening more clear.